### DISCUSSION TITLE
Who to read data from T01 file

### DISCUSSION THREAD

**Discussion by BastiLoe** [OpenRadioss/OpenRadioss#1498] [Q&A]
Hello all,

as of now, is there any chance to reaad data from T01 file except using commercial HyperGraph software?

Thanks BastiL

**Comment by MarianBulla** [comment]
Hello BastiLoe,

yes, it is possible. Just convert the T01 results into comma separated values.

See here: 
https://github.com/OpenRadioss/OpenRadioss/tree/main/tools/th_to_csv

Hope it helps you ?

Thanks and best regards,
Marian
